install/kubernetes/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -307,7 +307,7 @@ spec:
           {{- toYaml .Values.initResources | trim | nindent 10 }}
 {{- end }}
       restartPolicy: Always
-{{- if and (eq .Release.Namespace "kube-system") (or (gt .Capabilities.KubeVersion.Minor "10") (gt .Capabilities.KubeVersion.Major "1"))}}
+{{- if or (and (eq .Release.Namespace "kube-system") (gt .Capabilities.KubeVersion.Minor "10")) (gt .Capabilities.KubeVersion.Minor "16") (gt .Capabilities.KubeVersion.Major "1")}}
```

Comment:
@johngmyers this is not in the correct format. (same for all other changes)

It should be:
`{{- if or (and (eq .Release.Namespace "kube-system") (gt .Capabilities.KubeVersion.Minor "10")) (or (gt .Capabilities.KubeVersion.Minor "16") (gt .Capabilities.KubeVersion.Major "1"))}}`

Reply:
Fixed two more extaneous close parentheses.

The or operator in Go template is variadic, so the second or operator in your suggestion is unnecessary.